drivers/docker/config.go

```diff
@@ -551,6 +582,22 @@ func (d *Driver) SetConfig(c *base.Config) error {
 		d.config.GC.imageDelayDuration = dur
 	}
 
+	if len(d.config.GC.DanglingContainers.PeriodStr) > 0 {
+		dur, err := time.ParseDuration(d.config.GC.DanglingContainers.PeriodStr)
+		if err != nil {
+			return fmt.Errorf("failed to parse 'period' duration: %v", err)
+		}
+		d.config.GC.DanglingContainers.period = dur
```

Comment:
On second thought perhaps making 1m the minimum is desirable to ensure `Restore` has time to run?

This running before Restore completes is obviously very bad, so we should do everything we can to avoid it.